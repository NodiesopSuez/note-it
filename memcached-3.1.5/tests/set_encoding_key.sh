#!/bin/sh

/usr/local/Cellar/php@7.4/7.4.20/bin/php  -n -c '/Applications/XAMPP/xamppfiles/htdocs/note-it/memcached-3.1.5/tmp-php.ini'  -d "output_handler=" -d "open_basedir=" -d "disable_functions=" -d "output_buffering=Off" -d "error_reporting=32767" -d "display_errors=1" -d "display_startup_errors=1" -d "log_errors=0" -d "html_errors=0" -d "track_errors=0" -d "report_memleaks=1" -d "report_zend_debug=0" -d "docref_root=" -d "docref_ext=.html" -d "error_prepend_string=" -d "error_append_string=" -d "auto_prepend_file=" -d "auto_append_file=" -d "ignore_repeated_errors=0" -d "precision=14" -d "memory_limit=128M" -d "log_errors_max_len=0" -d "opcache.fast_shutdown=0" -d "opcache.file_update_protection=0" -d "opcache.revalidate_freq=0" -d "zend.assertions=1" -d "zend.exception_ignore_args=0" -d "extension_dir=/Applications/XAMPP/xamppfiles/htdocs/note-it/memcached-3.1.5/modules/" -d "extension=memcached.so" -d "session.auto_start=0" -d "tidy.clean_output=0" -d "zlib.output_compression=Off" -d "mbstring.func_overload=0" -f "/Applications/XAMPP/xamppfiles/htdocs/note-it/memcached-3.1.5/tests/set_encoding_key.php"  2>&1
