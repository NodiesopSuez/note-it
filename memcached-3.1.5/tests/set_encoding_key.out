Fatal error: Uncaught Error: Call to a member function setEncodingKey() on null in /Applications/XAMPP/xamppfiles/htdocs/note-it/memcached-3.1.5/tests/set_encoding_key.php:6
Stack trace:
#0 {main}
  thrown in /Applications/XAMPP/xamppfiles/htdocs/note-it/memcached-3.1.5/tests/set_encoding_key.php on line 6